package.json

```diff
@@ -5,6 +5,7 @@
     "lerna": "^3.20.2"
   },
   "scripts": {
+    "bump-version": "lerna version --force-publish --no-git-tag-version --yes",
```

Comment:
This is to upgrade the versions, then we need to create a PR

*Usage*
```
yarn bump-version minor
```

*Details*
`--force-publish`: bump version in all lerna packages (even without change)
`--no-git-tag-version`: doesn't create tag and doesn't push to git
`--yes`: no prompt